src/main.ts

```diff
@@ -168,6 +168,33 @@ export class Main {
       this.triggerUpdateCheck();
       this.setupUpdateInterval();
     }
+
+    // Make sure we have a ~/.chef directory
+    this.createChefDir();
+  }
+
+  // make the ~/.chef directory if it doesn't exit and add a sample credentials file
+  private createChefDir() {
+    let os = require('os'),
+    fs = require('fs'),
+    path = require('path');
+    let chefDir = os.homedir() + path.sep + '.chef';
+
+    if (!fs.existsSync(chefDir)) {
+        console.log("Creating the .chef dir at " + chefDir)
+        fs.mkdirSync(chefDir, 0o700);
+
+        let credentialsFile = chefDir + path.sep + "credentials";
+        let credentialContent = "# This is the Chef Infra credentials file used by the knife CLI and other tools\n" +
```

Comment:
It would be cool if we shipped this in file form, maybe in /opt/chef-workstation/templates.  Then we could copy the file in here, and also in the main-chef-wrapper which initializes this directory without a sample template.  

Thoughts?

Reply:
I want this to be very short lived. The idea is that this acts as a stepping stone to a wizard based setup that builds a real credentials file for the user at first run. In addition to that we'd add the ability to manage the profiles in the preferences for the app. We'd have to either duplicate that logic on the CLI side or share some logic with things like common templates.